apm-agent-core/src/main/java/co/elastic/apm/agent/impl/transaction/TraceContext.java

```diff
@@ -56,23 +58,6 @@
  *       Header name     Version           Trace-Id                Span-Id     Flags
  * </pre>
  * <p>
- * Binary representation (e.g. 0.11.0.0+ Kafka record header), based on
```

Comment:
This comment is still useful since we're supporting the backward compatible header